src/openrct2/object/RideObject.cpp

```diff
@@ -217,8 +217,7 @@ void RideObject::Load()
 
             if (vehicleEntry->car_visual != VEHICLE_VISUAL_RIVER_RAPIDS)
             {
-                const auto numRotationFrames = vehicleEntry->GetNumRotationFrames();
-                vehicleEntry->NumRotationFrames = numRotationFrames;
+                const auto numRotationFrames = vehicleEntry->NumRotationFrames();
```

Comment:
Accidental reversion?

Reply:
Intentional, although it is ironic this line almost completely goes back to how it used to be.